client/src/components/dashboard/campaign-admin-box/campaign-admin-box.html

```diff
@@ -0,0 +1,41 @@
+<md-card
+    ng-if="$ctrl.campaign.id"
+    ng-class="{'md-card--first' : $ctrl.index % 3 === 0, 'md-card--last' : $ctrl.index % 3 === 2}"
+    ui-sref="main.campaign({id: $ctrl.link})">
+  <h3 class="campaign__name md-title">{{ ::$ctrl.campaign.name }}</h4>
+  <span class="campaign__dates">{{ ::$ctrl.campaign.open_date | date }} - {{ ::$ctrl.campaign.close_date | date }}</span>
+
+  <div class="camapign__round"
+      layout="column" layout-align="start stretch">
+    <h4>Latest Round</h4>
+    <span ng-if="$ctrl.lastRound">
+      {{ ::$ctrl.lastRound.number | ordinal }} Round ·
+      {{ ::("round." + $ctrl.lastRound.round.vote_method) | translate }} · 
+      <span class="muted">{{ ::$ctrl.lastRound.round.status }}</span>
+    </span>
+    <span class="muted" ng-if="!$ctrl.lastRound">No Rounds</span>
+  </div>
+  <div class="campaign__coordinators">
+    <h4>Coordinators ({{ ::$ctrl.campaign.coordinators.length }})</h4>
+    <div class="campaign__coordinators-list"
+        layout="row" layout-align="start start">
+      <span class="round__juror round__juror--small"
+            ng-repeat="juror in $ctrl.campaign.coordinators | orderBy: 'username'">
+          <md-tooltip>{{ ::juror.username }}</md-tooltip>
+          <span class="juror__avatar"
+              mont-avatar="{{ ::juror.username }}">
+            {{ ::juror.username[0].toUpperCase() }}
+          </span>
+      </span>
+    </div>
+  </div>
+</md-card>
+
+<md-card class="md-card--new"
+    layout="column" layout-align="center center"
+    ng-if="$ctrl.campaign === 'new'">
```

Comment:
Any user with `is_organizer == True` should be able to create campaigns. Right now, it looks like I can only create a campaign when there are other existing campaigns.